<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageEditor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/logo/win.png</normaloff>:/new/logo/win.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1181</width>
      <height>831</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>35</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="undoLayer">
     <item>
      <widget class="QPushButton" name="btnUndo">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Asset/Resource.qrc">
         <normaloff>:/Icon/Icon/undo.svg</normaloff>:/Icon/Icon/undo.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>840</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>840</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>35</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="rotate270Layer">
     <item>
      <widget class="QPushButton" name="btnRotate270">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Asset/Resource.qrc">
         <normaloff>:/Icon/Icon/left.svg</normaloff>:/Icon/Icon/left.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>35</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="rotate90Layer">
     <item>
      <widget class="QPushButton" name="btnRotate90">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Asset/Resource.qrc">
         <normaloff>:/Icon/Icon/right.svg</normaloff>:/Icon/Icon/right.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>35</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="rotate180Layer">
     <item>
      <widget class="QPushButton" name="btnRotate180">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Asset/Resource.qrc">
         <normaloff>:/Icon/Icon/rotate180.svg</normaloff>:/Icon/Icon/rotate180.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuBasic">
    <property name="title">
     <string>Basic</string>
    </property>
    <widget class="QMenu" name="menuRotate">
     <property name="title">
      <string>Rotate</string>
     </property>
     <addaction name="actionRotate_90"/>
     <addaction name="actionRotate_270"/>
     <addaction name="actionRotate_180"/>
    </widget>
    <addaction name="menuRotate"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionUndo"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="actionGray"/>
    <addaction name="actionBlur"/>
    <addaction name="actionSharp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBasic"/>
   <addaction name="menuFilter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="text">
    <string>Blur</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionSharp">
   <property name="text">
    <string>Sharp</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRotate_90">
   <property name="text">
    <string>Rotate 90°</string>
   </property>
  </action>
  <action name="actionRotate_270">
   <property name="text">
    <string>Rotate -90°</string>
   </property>
  </action>
  <action name="actionRotate_180">
   <property name="text">
    <string>Rotate 180°</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Asset/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
